build/chip/chip_test_suite.gni

```diff
@@ -12,12 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+enable_efr32_build_per_testsource = true  #++++ TODO: Figure out how to enable things based on `chip_device_platform == "efr32"` which is not currently in scope.
+
 import("//build_overrides/build.gni")
 import("//build_overrides/chip.gni")
+if (enable_efr32_build_per_testsource) {
+  import("//build_overrides/efr32_sdk.gni")  # Defines efr32_sdk_build_root
+  import("//build_overrides/nlunit_test.gni")  # Defines nlunit_test_root
+}
 
 import("${chip_root}/build/chip/chip_test.gni")
 import("${chip_root}/build/chip/tests.gni")
 import("${dir_pw_unit_test}/test.gni")
+if (enable_efr32_build_per_testsource) {
+  import("${efr32_sdk_build_root}/efr32_sdk.gni")
+  import("${efr32_sdk_build_root}/silabs_executable.gni")
+  #import("$dir_pw_protobuf_compiler/proto.gni")
+
+  #++++ TODO: Figure out how to get access to these variables from src/test_driver/efr32/BUILD.gn so we don't have to set them explicitly here.
+  examples_plat_dir = "${chip_root}/examples/platform/silabs/efr32"
+  examples_common_plat_dir = "${chip_root}/examples/platform/silabs"
+  #efr32_project_dir = "${chip_root}/src/test_driver/efr32"
+}
```

Comment:
A bunch of stuff needs importing into here since it is used by `silabs_executable`.  Eventually I'll find a cleaner way to do this.